contrib/epee/src/readline_buffer.cpp

```diff
@@ -16,6 +17,8 @@ static std::string last_prompt;
 std::mutex line_mutex, sync_mutex, process_mutex;
 std::condition_variable have_line;
 
+std::vector<std::string> rdln::readline_buffer::completion_commands = {"exit"};
```

Comment:
The readline patches have been adding lots to RW static memory. This in particular has a initialization order fiasco (that we should be unlikely to hit, but its easy to oops that).

I am wondering if it makes sense to refactor all readline static variables in one class, and then have that class constructed as a function local static. It would be slightly slower since each access to a function local static now requires thread synchronization, but the delay would be marginal and the code would be more resilient to crazy C++ behavior if the functions were used elsewhere for some reason.